docs/docs/tutorial.md

```diff
@@ -549,7 +549,7 @@ var CommentForm = React.createClass({
     var author = this.state.author.trim();
     var text = this.state.text.trim();
     if (!text || !author) {
-      return;
+      return false;
```

Comment:
@jimfb @Aweary The newest version fixes the issue. I just submitted a PR updating the example's React dependency.